testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/core/basic/CacheControlAnnotationTest.java

```diff
@@ -60,14 +66,14 @@ public void after() throws Exception {
     @Test
     public void testResourceValid() {
         WebTarget base = client.target(generateURL("/maxage"));
-        Response response = base.request().get();
-
-        Assert.assertEquals(HttpServletResponse.SC_OK, response.getStatus());
-        CacheControl cc = CacheControl.valueOf(response.getHeaderString("cache-control"));
-        Assert.assertFalse("Cache should not be private", cc.isPrivate());
-        Assert.assertEquals("Wrong age of cache", 3600, cc.getMaxAge());
 
-        response.close();
+        try (Response response = base.request().get()) {
+            Assert.assertEquals(HttpServletResponse.SC_OK, response.getStatus());
+            CacheControl cc = RuntimeDelegate.getInstance().createHeaderDelegate(CacheControl.class)
+                    .fromString(response.getHeaderString("cache-control"));
+            Assert.assertFalse("Cache should not be private", cc.isPrivate());
+            Assert.assertEquals("Wrong age of cache", 3600, cc.getMaxAge());
+        }
```

Comment:
This is just an FYI. While I don't mind changes like this, they should be in a separate commit. In this specific case, I don't think a JIRA is needed. You don't need to make that change for this PR, but in the future changes for formatting, patterns or removing deprecated code uses should be done separate from the main change being performed.